doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -1371,7 +1371,9 @@ Abbreviations
 
    For instance, suppose that we define the following.
 
-   :n:`Ltac2 Notation foo := fun x => x ().`
+   .. rocqdoc::
```

Comment:
Note that `.. rocdoc::` always inserts an extra blank line, which doesn't look right.

![image](https://github.com/user-attachments/assets/980491c8-369d-410e-bdc5-cd6e666dc4f2)